SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -608,11 +608,18 @@ int64 USpatialActorChannel::ReplicateActor()
 
 			bCreatedEntity = true;
 
-			// Since we've tried to create this Actor in Spatial, we no longer have authority over the actor since it hasn't been delegated to us.
-			if (!USpatialStatics::IsSpatialOffloadingEnabled())
+			// If we're spawning an Actor that we know will be load-balanced to another worker,
+			// then preemptively set the role to SimulatedProxy.
+			if (SpatialGDKSettings->bEnableUnrealLoadBalancer)
 			{
-				Actor->Role = ROLE_SimulatedProxy;
-				Actor->RemoteRole = ROLE_Authority;
+				const UAbstractLBStrategy* LBStrategy = NetDriver->LoadBalanceStrategy;
+				if (!LBStrategy->ShouldHaveAuthority(*Actor))
+				{
+					Actor->Role = ROLE_SimulatedProxy;
```

Comment:
We should switch roles if not using offloading or load balancer also.

Reply:
If we're not using offloading or load-balancer, i.e. single-server, don't we want to keep auth since we know we're getting it?

Reply:
We _could_ be running multiple servers without either of those options on - runtime would load balance. Until we default on the load balancer, we should support both paths.